edit-post/components/header/style.scss

```diff
@@ -25,6 +25,7 @@
 
 	@include break-medium() {
 		top: $admin-bar-height;
+		left: $admin-sidebar-width;
```

Comment:
Should this be included in the `editor-left` mixin instead?

Reply:
You can update this pull request ;)

Reply:
Thanks